Updated web links for same54_xpro

robertperkel opened the pull request:
doc: same54_xpro fixed web links

The weblinks to microchip were broken. Links have been updated.
Fixes #88009

robertperkel committed:
doc: atmel: same54_xpro Fixed broken links in index

doc: same54_xpro Fixed broken links in index

- Updated pdf link to datasheet
- Added html link to datasheet
- Added link to product page
- Updated zip file link
- Added zip file link to newer revision

Signed-off-by: Robert Perkel <robert.perkel@microchip.com>

nandojve commented on boards/atmel/sam0/same54_xpro/doc/index.rst:
```diff
@@ -7,7 +7,7 @@ The SAM E54 Xplained Pro evaluation kit is ideal for evaluation and
 microcontrollers. The kit includes Atmel’s Embedded Debugger (EDBG),
 which provides a full debug interface without the need for additional
-hardware.
+hardware. 
```
remove space after dot

nandojve commented on boards/atmel/sam0/same54_xpro/doc/index.rst:
```diff
@@ -160,10 +160,16 @@ References
+.. _SAM E54 Xplained Pro Schematic (Blue PCB):
+    https://ww1.microchip.com/downloads/aemDocuments/documents/OTH/ProductDocuments/BoardDesignFiles/SAM-E54-Xplained-Pro-Design-Documentation-Rev9.zip
+
+.. _SAM E54 Xplained Pro Schematic (Red PCB):
+   https://ww1.microchip.com/downloads/aemDocuments/documents/OTH/ProductDocuments/BoardDesignFiles/SAM-E54-Xplained-Pro-Design-Documentation-Rev11.zip 
```
Without a reference in the text this link won't be rendered to the user.

nandojve commented on boards/atmel/sam0/same54_xpro/doc/index.rst:
```diff
@@ -160,10 +160,16 @@ References
 
 .. _Microchip website:
-    http://www.microchip.com/DevelopmentTools/ProductDetails.aspx?PartNO=ATSAME54-XPRO
+    https://www.microchip.com/en-us/development-tool/ATSAME54-XPRO
```
same with this.

robertperkel commented:
Added references to the other links and removed the extraneous space

pdgendt commented:
> Added references to the other links and removed the extraneous space

Please read the [guidelines](https://github.com/zephyrproject-rtos/zephyr/pull/88128), as the bot stated, on how to format your commits messages.
Additionally your commits need to be squashed into a single one, as we don't accept fix-up commits in a single PR.

robertperkel commented:
Squashed previous commits and updated rst file formatting to match guidelines. 

Does anyone have a recommendation for a good file previewer for this file type? I'm not having a lot of luck finding a good solution on Windows

nandojve commented:
> Does anyone have a recommendation for a good file previewer for this file type? I'm not having a lot of luck finding a good solution on Windows

My recommendation is that you build the docs and run the python server to have access in the web browser.
see [documentation/generation](https://docs.zephyrproject.org/latest/contribute/documentation/generation.html)

robertperkel reviewed (commented):
- Fixed broken links to microchip
- Added new links for alternative schematics / datasheet format (html)
